### PR TITLE
Update package_krb5-server_removed

### PR DESCRIPTION
**Pull request by Xeicker**
#### Description:

- Update the rule with DISA's content as this is not used by any other product/profile. Select the rule for ol8 and rhel8 STIG profiles

#### Rationale:

- This rule complies with OL08-00-010163 & RHEL-08-010163

### CODE REVIEW

File: products/rhel8/profiles/stig.profile [hunk 1]
```diff
@@ -1173,3 +1173,6 @@ selections:
 
     # RHEL-08-040390
     - package_tuned_removed
+
+    # RHEL-08-010163
+    - package_krb5-server_removed
```
Comment (vojtapolasek) [inline products/rhel8/profiles/stig.profile:1178]: Thank you very much for enhancing the RHEL8 profile as well.
We have some additional tests for RHEL profiles to ensure that they do not get modified by accident. We store current compiled version of the profile (without comments etc) in tests/data/profile_stability/rhel8/stig.profile.
Could you please do the following:
1. ./build_product rhel8
2. cp build/rhel8/profiles/stig.profile tests/data/profile_stability/rhel8/stig.profile
3. the same for stig_gui.profile cause it extends the stig profile
4. commit

Thank you.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello,
thank you very much for the contribution. Please see comments.

File: linux_os/guide/services/kerberos/package_krb5-server_removed/rule.yml [hunk 2]
```diff
@@ -5,20 +5,30 @@ prodtype: ol7,ol8,rhel7,rhel8,rhel9
 title: 'Remove the Kerberos Server Package'
 
 description: |-
-    The <tt>krb5-server</tt> package should be removed if not in use.
-    Is this system the Kerberos server? If not, remove the package.
-    {{{ describe_package_remove(package="krb5-server") }}}
```
[the remark this answers is not in the document]
Comment (Xeicker) [reply linux_os/guide/services/kerberos/package_krb5-server_removed/rule.yml]: I think no strong reason. I restored it as it was

File: linux_os/guide/services/kerberos/package_krb5-server_removed/rule.yml [hunk 3]
```diff
@@ -5,20 +5,30 @@ prodtype: ol7,ol8,rhel7,rhel8,rhel9
...
-rationale: |-
-    Unnecessary packages should not be installed to decrease the attack
-    surface of the system.  While this software is clearly essential on an KDC
-    server, it is not necessary on typical desktop or workstation systems.
+    krb5-server.x86_64 1.17-9.el8 repository
+    </pre>
 
-severity: low
+rationale: |-
+    Unapproved mechanisms that are used for authentication to the cryptographic
```
[the remark this answers is not in the document]
Comment (Xeicker) [reply linux_os/guide/services/kerberos/package_krb5-server_removed/rule.yml]: I think no strong reason. I restored it as it was

**Comment by vojtapolasek** [comment]
Hello, thank for fixes. I see you took the CCE from the cce-redhat-avail.txt. But you have to actually use the cce by putting the 
```
identifiers:
    cce@rhel8: <PUT CCE HERE>
```
into the rule.yml file for the rule. I am sorry if I did not explain this clearly before.

**Comment by Xeicker** [comment]
Apparently I restored too much in rule.yml, sorry

**Comment by vojtapolasek** [comment]
/retestHello, thank for fixes. I see you took the CCE from the cce-redhat-avail.txt. But you have to actually use the cce by putting the 
```
identifiers:
    cce@rhel8: <PUT CCE HERE>
```
into the rule.yml file for the rule. I am sorry if I did not explain this clearly before.

**Review by vojtapolasek — APPROVED** [review]
Looks good now, thank you.

**Pull request merged**